src/analytics/src/main/scala/osmesa/analytics/oneoffs/StreamingAOIMonitor.scala

```diff
@@ -315,17 +307,132 @@ object AOIMonitorUtils extends Logging{
     }
   }
 
-  def getLastSequence(databaseURI: URI, procName: String)(implicit spark: SparkSession): Option[Int] = {
-    spark.sparkContext.parallelize(Seq(databaseURI)).map { uri =>
-      queryLastSequence(uri, procName)
-    }.collect.apply(0)
+  def setBeginSequence(interval: Interval, lastSequence: Int) = {
+    var connection: Connection = null
+    try {
+      connection = AOIDatabaseConfig.getConnection
+      val preppedStatement =
+        connection.prepareStatement("INSERT INTO checkpoint_interval VALUES (?, ?)")
+      preppedStatement.setString(1, interval.value)
+      preppedStatement.setLong(2, lastSequence)
+      preppedStatement.executeQuery()
+    } catch {
+      case e: Throwable => {
+        println(s"ERROR in setBeginSequence: ${e.getMessage}")
```

Comment:
This should probably send a comprehensive error message to the logger since this will be the only way we have to identify a problem with the checkpoint setting.  I.e., in case there is a problem with the run and we have to go back and manually update the checkpoint table, we should know what the sequence number is and for which interval (mostly to minimize the possibility of a screw-up).

Reply:
👍 I'll swap back to the helper warning function you added in this script.